four-node-cluster/setup/resources/createDB.sql

```diff
@@ -0,0 +1,12 @@
+create database IDENTITY_DB;
```

Comment:
why can't we use existing scripts

Reply:
arapt from the yml file other stuff should be able to dupliccate

Reply:
Addressed by https://github.com/wso2/performance-is/pull/128/commits/20e9c6ecdc067d08d3d4dade6fcb8d9f2a6e9d37